iTechArt.Api/Controllers/StudentsController.cs

```diff
@@ -34,22 +34,18 @@ public async Task<ActionResult> ImportAsync(IFormFile file)
                     await _studentsService.ImportStudentsAsync(file);
                     return Ok();
                 }
-
-                return BadRequest("Invalid file format!");
-            }
-            else
-            {
-                return BadRequest("Invalid file format!");
             }
+             
+            return BadRequest("Invalid file format!");
         }
 
         /// <summary>
         /// Get all students
         /// </summary>
         [HttpGet("get_all")]
-        public async Task<ActionResult<IStudent[]>> GetAllAsync()
+        public async Task<ActionResult<IStudent[]>> GetAllAsync([FromQuery] int pageIndex)
```

Comment:
if UI what to change paging count elements, they are can`t do it.

Reply:
We decided to make it constant
If we need, we may make some changes